src/main/java/com/criteo/vips/VipsImageImpl.java

```diff
@@ -57,6 +57,21 @@ public VipsImageImpl(VipsImageImpl image, PixelPacket color) throws VipsExceptio
         newFromImage(image, color);
     }
 
+    private VipsImageImpl(int width, int height) throws VipsException {
+        blackNative(width, height);
+    }
+    private native void blackNative(int width, int height) throws VipsException;
+
+    /** Make a black unsigned char image of a specified size
+     *
+     * @param width output width
+     * @param height output height
+     * @throws VipsException
+     */
```

Comment:
Please declare and move comment  it in VipsImage.java

Reply:
you cannot put code in an interface